src/Uno.UI.Toolkit/Diagnostics/DiagnosticsOverlay.cs

```diff
@@ -381,7 +381,12 @@ protected override void OnApplyTemplate()
 			//_anchor.Tapped += OnAnchorTapped;
 			_anchor.ManipulationDelta += OnAnchorManipulated;
 			_anchor.ManipulationCompleted += OnAnchorManipulatedCompleted;
+#if !__SKIA__
 			_anchor.ManipulationMode = ManipulationModes.TranslateX | ManipulationModes.TranslateY | ManipulationModes.TranslateInertia;
```

Comment:
It would be interesting to make this conditional to the availability of TranslateInertia. There's likely a ApiInformation call that we can make to have enabled at runtime.

Reply:
I see we have this: https://github.com/unoplatform/uno/blob/a25a2f54454e990a6865e9ea7c6a2646c9ba508f/src/Uno.UI/UI/Xaml/Input/ManipulationModes.cs#L50-L59

But TranslateInertial itself isn't tagged with [`Uno.NotImplemented]` here: https://github.com/unoplatform/uno/blob/a25a2f54454e990a6865e9ea7c6a2646c9ba508f/src/Uno.UI/UI/Xaml/Input/ManipulationModes.cs#L37

Reply:
@jeromelaban Should we be using this `IsSupported()` method or instead make an `ApiInformation` call?